meta-oe/recipes-connectivity/openssh/openssh_%.bbappend

```diff
@@ -0,0 +1,14 @@
+do_install:append () {
+  if ${@bb.utils.contains("DISTRO_FEATURES", "hiba", "false", "true", d)} ; then
+    echo "DISTRO_FEATURES does not support hiba. Skip the installations"
+    exit 0
+  fi
```

Comment:
this maybe redundant.

Reply:
rm'ed and will be moved to poky recipe once I get write access.